### DISCUSSION TITLE
[초보주의] 유니티에 본인 제페토 캐릭터를 넣어서 테스트 하고싶어요. 어떻게하나요?

### DISCUSSION THREAD

**Discussion by sudaengman** [naverz/zepeto-studio-kor#523] [5. Q&A]
본인 코딩 경험은 html, css, 자바스크립트 약간 배웠구용..
제페토월드를 만들어보고 싶은 늅뉴비입니다..ㅜ
유니티 경험 x입니다....ㅜㅜㅜ

이것저것 실험해보고 싶어서 
열심히 유튜브 보면서 월드를 만들었어요

![image](https://user-images.githubusercontent.com/102212003/159644098-346515ef-f9f3-4f61-81f5-f32291379770.png)

근데 플레이를 누르면 

![image](https://user-images.githubusercontent.com/102212003/159644249-ffc3a868-7d24-4f0b-9104-f4a1ac2e8398.png)

ㅇ이런화면이 뜨고 캐릭터는 없더라구요...

그래서 캐릭터를 넣어보려고 이것저것 눈팅을 시도헀어요.
" https://github.com/naverz/zepeto-studio-kor/discussions/246 "
여기서 보면 캐릭터 리스폰 방법이 자세하게 나와있더라구요,

근데 캐릭터 리스폰을 하긴 하는데, '어떤 캐릭터를 리스폰을 할 것이냐' 를 생각해보니 머리가 아파오더라구요..
아무튼 저의 제페토 캐릭터를 넣어서 테스트해보고 싶은데..
어떻게 캐릭터를 불러와서 움직이게 할 수 있을까요?

**Comment by x4cf** [comment]
제페토 멀티 셈플을 참고하기를 추천합니다.
아주 간단하게 플레이하는 유저에 맞게 유저 자신의 캐릭터를 가져올 수 있어요 :D

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 다음 가이드에 있는 내용을 똑같이 따라하면 내 캐릭터가 화면에 나타납니다.
https://studio.zepeto.me/kr/guides/character-control
